spec/integration_spec.rb

```diff
@@ -67,10 +67,6 @@ module TestData
       expect(document.usable_tags.count).to be(30)
     end
 
-    it 'has the expected amount of role tags' do
-      expect(document.role_tags.count).to be(6)
```

Comment:
Why remove this test?

Reply:
Because I removed the method from the `Document` class.